File: packages/cli/hieudoanm/rust/src/cmd/blackjack.rs
pub fn command() -> clap::Command {
    clap::Command::new("blackjack")
        .about("Blackjack card counting (TUI)")
}

pub fn run(_matches: &clap::ArgMatches) -> anyhow::Result<()> {
    println!("blackjack card counting (TUI)");
    Ok(())
}

File: packages/cli/hieudoanm/swift/Sources/hieudoanm/Commands/Blackjack.swift
import Foundation
import ArgumentParser

struct Blackjack: ParsableCommand {
    static let configuration = CommandConfiguration(
        commandName: "blackjack",
        abstract: "Card counting practice"
    )

    mutating func run() {
        print("Blackjack TUI not available in Swift version.")
        print("Run the Go binary for the interactive card counting experience.")
    }
}
